webapp/survey/nodeDefs/actions.js

```diff
@@ -2,29 +2,58 @@ import axios from 'axios'
 import * as R from 'ramda'
 
 import { uuidv4 } from '@core/uuid'
-
-import { debounceAction } from '@webapp/utils/reduxUtils'
-import { appModuleUri, designerModules } from '@webapp/loggedin/appModules'
-
 import * as Survey from '@core/survey/survey'
 import * as NodeDef from '@core/survey/nodeDef'
 import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+import * as SurveyValidator from '@core/survey/surveyValidator'
 import * as NodeDefValidations from '@core/survey/nodeDefValidations'
 
-import * as AppState from '@webapp/app/appState'
+import { debounceAction } from '@webapp/utils/reduxUtils'
 
-import { showNotification } from '@webapp/app/appNotification/actions'
+import { appModuleUri, designerModules } from '@webapp/loggedin/appModules'
+
+import * as AppState from '@webapp/app/appState'
 import * as NotificationState from '@webapp/app/appNotification/appNotificationState'
 import * as SurveyState from '../surveyState'
+import * as NodeDefState from '@webapp/loggedin/surveyViews/nodeDef/nodeDefState'
+
+import { hideAppLoader, showAppLoader } from '@webapp/app/actions'
+import { showNotification } from '@webapp/app/appNotification/actions'
+import { nodeDefEditUpdate } from '@webapp/loggedin/surveyViews/nodeDef/actions'
 
 export const nodeDefCreate = 'survey/nodeDef/create'
-export const nodeDefPropsUpdate = 'survey/nodeDef/props/update'
+export const nodeDefUpdate = 'survey/nodeDef/update'
 export const nodeDefDelete = 'survey/nodeDef/delete'
+export const nodeDefPropsUpdateTemp = 'survey/nodeDef/props/update/temp'
```

Comment:
why temp postfix?

Reply:
I wanted to give enphasys to the fact that the props are not updated immediately to the db, but I can remove it.